tests/baselines/reference/collisionArgumentsClassConstructor.errors.txt

```diff
@@ -42,63 +91,113 @@ tests/cases/compiler/collisionArgumentsClassConstructor.ts(61,17): error TS2396:
         constructor(public arguments: number, ...restParameters) { //arguments is error
                     ~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2396: Duplicate identifier 'arguments'. Compiler uses 'arguments' to initialize rest parameters.
+                           ~~~~~~~~~
+!!! error TS1100: Invalid use of 'arguments' in strict mode.
             var arguments = 10; // no error
+                ~~~~~~~~~
+!!! error TS1100: Invalid use of 'arguments' in strict mode.
         }
     }
     class c3NoError {
         constructor(public arguments: number) { // no error
+                           ~~~~~~~~~
+!!! error TS1100: Invalid use of 'arguments' in strict mode.
             var arguments = 10; // no error
+                ~~~~~~~~~
+!!! error TS1100: Invalid use of 'arguments' in strict mode.
         }
     }
     
     declare class c4 {
         constructor(i: number, ...arguments); // No error - no code gen
+                                  ~~~~~~~~~

```

Comment:
That's interesting.  We haven't ever distinguished ambient/non-ambient parsing in that manner.

I think it might be best if we kept hte behavior hte same regardless of if this was ambient or not.  But i could be convinced otherwise.